### PR TITLE
QML tablet layout progress

### PR DESCRIPTION
**Pull request by fayeli**
I recreated Alan's design in QML, laying out the tablet buttons and the audio bar. I also created hover and click state for the mute button. It's not bind to the actual mic muting behavior yet.
 
Screenshots:
image
image

I've also tested it in Interface, and it looks pretty good! Also, I've noted that other people around cannot see what's on my tablet. They can only see a blank white screen. That's expected behaviour, right?

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: e38593bce7ae36879dc4c66cc2c879a4b6a80945 available at: https://gist.github.com/9b9716211d112a1cbfa609f32045ebd8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-e38593b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-e38593b.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e38593bce7ae36879dc4c66cc2c879a4b6a80945 available at: https://gist.github.com/9b9716211d112a1cbfa609f32045ebd8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-e38593b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-e38593b.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 385eadce94db53d2cb48cf2aacdae414e5aae3f3 available at: https://gist.github.com/9b9716211d112a1cbfa609f32045ebd8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-385eadc.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-385eadc.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 785c66bfb595cce1065f57e57327ac9fb4f75e37 available at: https://gist.github.com/9b9716211d112a1cbfa609f32045ebd8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-785c66b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9193-785c66b.dmg

File: interface/resources/qml/hifi/tablet/TabletButton_Green.qml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.0
```
Comment (hyperlogic) [inline interface/resources/qml/hifi/tablet/TabletButton_Green.qml]: If the only difference between TabletButton_Green.qml and TabletButton.qml is the color.  You can use qml's inheritance to only override the color.  You can do this by replacing this qml with the following:

    TabletButton {
        color: "#1fc6a6"
    }

Or you can explicitly change the color when instancing a TabletButton in Tablet.qml

    var component_a = Qt.createComponent("TabletButton.qml");
    var obj = component_a.createObject(flowMain);
    obj.color = "#1fc6a6" // green

I'd rather avoid adding extra qml files, if we can avoid it.

Comment (fayeli) [inline interface/resources/qml/hifi/tablet/TabletButton_Green.qml]: Got it. I'm now working on using more of qml's inheritance for the buttons on my new branch

**Comment by fayeli** [comment]
Close PR because my work should live in a different branch

**Pull request closed without merging**